### PR TITLE
Affrae's Changes

### LINKED ISSUE #13: Affrae to add some stuff
- [x] A file created in browser
- [x] A file created locally

### PR DESCRIPTION
**Pull request by affrae**
Closes #13 

Added some files - @githubteacher :eyes: at this?

### CODE REVIEW

File: Affrae - Things to do in the mountains.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+### Fun things to do in the blue mountains.
+
+Do yourself a favour - Go Here:
+
+
+![Wentworth Falls](https://assets.atdw-online.com.au/images/6d5b3c76bfe2ddef60985d61325212b0.jpeg?rect=157,0,2048,1536&w=800&h=600)
```
Comment (githubteacher) [inline Affrae - Things to do in the mountains.md:8]: @affrae this looks awesome :sparkles:

File: From Affrae/affrae-file-created-in-browser.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file was created in the web UI!
```
Comment (githubteacher) [inline From Affrae/affrae-file-created-in-browser.md:1]: Great job :taco:

**Review by githubteacher — APPROVED** [review]
This is looking good so far @affrae

File: Affrae - Things to do in the mountains.md (hunk 1 above)
Comment (ghost) [inline Affrae - Things to do in the mountains.md:8]: @affrae Can you give an address?

File: From Affrae/affrae-file-created-locally.md [hunk 3]
```diff
@@ -0,0 +1 @@
+This file was created in Atom locally on **Affrae's computer**!
```
Comment (ghost) [inline From Affrae/affrae-file-created-locally.md]: This looks like a stub.

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by affrae** [comment]
@jgancisra I added a google maps location is that ok?

File: From Affrae/affrae-file-created-locally.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+No longer a stub!: This file was created in Atom locally on **Affrae's computer**!
```
Comment (githubteacher) [inline From Affrae/affrae-file-created-locally.md:4]: You need to change this

**Review by githubteacher — CHANGES_REQUESTED** [review]

**Review by githubteacher — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**